### ISSUE TITLE
Request command line argument for program file path

### ISSUE THREAD

**Issue by MCPEmaniac** [commanderx16/x16-emulator#50]
I recommend consideration be given to adding a command line argument for a path to the program files which makes it easier to store them in a directory other than where the executable resides.

**Comment by rsbohn12** [comment]
Perhaps `-datasette /home/pi/x16-programs`? The default path is the current directory, which doesn't have to be where the executable resides. It does, however, have to have the rom.bin and chargen.bin unless you provide the startup arguments for them.

**Comment by MCPEmaniac** [comment]
Yes something like that, but perhaps -prog instead of -datasette.

**Comment by bcbock** [comment]
I think calling it sd might be more appropriate. Also, I can’t figure out how to create a blank SD card image. Perhaps you could include one, or instructions how to create one. (I’m on a Mac, so I can’t seem to create a fat32 image, just fat or exfat.

**Comment by greg-king5** [comment]
`-prgdir` is another choice.
